pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1143,6 +1145,9 @@ class Timedelta(_Timedelta):
     Notes
     -----
     The ``.value`` attribute is always in ns.
+
+    If the precision is higher than nanoseconds, the precision of the duration is
+    truncated to nanoseconds.
```

Comment:
clarify this is for a string input

do additional digits round or just drop (and if just drop, are we sure thats the desired behavior?)

Reply:
Dropd additional digits. Was not sure either so I started with the easier implementation
https://github.com/pandas-dev/pandas/pull/36771#issuecomment-706768447